From 1f7f2beb8a94133938891fdff1fdd179603dd120 Mon Sep 17 00:00:00 2001
From: "M. Vefa Bicakci" <vefa.bicakci@windriver.com>
Date: Wed, 27 Oct 2021 13:38:14 -0400
Subject: [PATCH] libnftnl.spec: Modify Release field for StarlingX

This commit modifies the Release field of the libnftnl package's spec
file for StarlingX.

Signed-off-by: M. Vefa Bicakci <vefa.bicakci@windriver.com>
---
 SPECS/libnftnl.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SPECS/libnftnl.spec b/SPECS/libnftnl.spec
index 4769895895b7..19681de0c2af 100644
--- a/SPECS/libnftnl.spec
+++ b/SPECS/libnftnl.spec
@@ -1,5 +1,5 @@
 %define rpmversion 1.1.5
-%define specrelease 4%{?dist}
+%define specrelease 4%{?_tis_dist}.%{tis_patch_ver}
 
 Name:           libnftnl
 Version:        %{rpmversion}
-- 
2.29.2

